### ISSUE TITLE
BaseRecalibrator regression

### ISSUE THREAD

**Issue by jean-philippe-martin** [broadinstitute/hellbender#866]
Somewhere between #835 and now, BaseRecalibrator stopped working. When I try to run testBQSRBucket, I get the error below. This test is currently enabled so regression tests should have caught this.

````
java.lang.RuntimeException: java.lang.RuntimeException: java.lang.NoSuchMethodError: com.google.common.base.Stopwatch.createStarted()Lcom/google/common/base/Stopwatch;
	at org.broadinstitute.hellbender.tools.IntegrationTestSpec.executeTest(IntegrationTestSpec.java:131)
	at org.broadinstitute.hellbender.tools.IntegrationTestSpec.executeTest(IntegrationTestSpec.java:104)
	at org.broadinstitute.hellbender.tools.IntegrationTestSpec.executeTest(IntegrationTestSpec.java:86)
	at org.broadinstitute.hellbender.tools.dataflow.pipelines.BaseRecalibratorDataflowIntegrationTest.testBQSRBucket(BaseRecalibratorDataflowIntegrationTest.java:176)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:639)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:821)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:124)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:773)
	at org.testng.TestRunner.run(TestRunner.java:623)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1185)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1110)
	at org.testng.TestNG.run(TestNG.java:1018)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
	at org.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:125)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:483)
	at com.intellij.rt.execution.application.AppMain.main(AppMain.java:140)
Caused by: java.lang.RuntimeException: java.lang.NoSuchMethodError: com.google.common.base.Stopwatch.createStarted()Lcom/google/common/base/Stopwatch;
	at com.google.cloud.dataflow.sdk.Pipeline.run(Pipeline.java:166)
	at org.broadinstitute.hellbender.engine.dataflow.DataflowCommandLineProgram.runPipeline(DataflowCommandLineProgram.java:145)
	at org.broadinstitute.hellbender.engine.dataflow.DataflowCommandLineProgram.doWork(DataflowCommandLineProgram.java:107)
	at org.broadinstitute.hellbender.cmdline.CommandLineProgram.runTool(CommandLineProgram.java:98)
	at org.broadinstitute.hellbender.cmdline.CommandLineProgram.instanceMain(CommandLineProgram.java:151)
	at org.broadinstitute.hellbender.Main.instanceMain(Main.java:71)
	at org.broadinstitute.hellbender.Main.instanceMain(Main.java:78)
	at org.broadinstitute.hellbender.CommandLineProgramTest.runCommandLine(CommandLineProgramTest.java:75)
	at org.broadinstitute.hellbender.tools.IntegrationTestSpec.executeTest(IntegrationTestSpec.java:126)
	... 33 more
Caused by: java.lang.NoSuchMethodError: com.google.common.base.Stopwatch.createStarted()Lcom/google/common/base/Stopwatch;
	at com.google.cloud.genomics.dataflow.readers.bam.Reader.process(Reader.java:93)
	at com.google.cloud.genomics.dataflow.readers.bam.ReadBAMTransform$ReadFn.processElement(ReadBAMTransform.java:68)
````

**Comment by jean-philippe-martin** [comment]
This contrasts with my local branch (#863, awaiting review) where testBQSRBucket passes (but testBQSRCloud fails).

**Comment by jean-philippe-martin** [comment]
David reports that the integration tests are still passing, so this issue appears to impact only me.

**Comment by lbergelson** [comment]
@jean-philippe-martin Are you seeing this issue in Intellij or on the command line?  I'm seeing a similar issue in intellij and it has to do with dependency ordering.  Intellij sometimes gets the ordering different than gradle and ends up with an old version of guava.

**Comment by jean-philippe-martin** [comment]
This was on intelliJ. Of course it's also important for tests to work there, since that's what I usually check. Adding the global ban on guava-jdk5 as I do in #867 appears to fix this problem for good.

**Comment by lbergelson** [comment]
I think the global ban is the right thing.  If you see problems like this in the future, only in intellij, you can edit the project structure manually to specify the dependency you meant.  It's a dumb hack, but it can save pain.

**Comment by jean-philippe-martin** [comment]
Thanks!

**Comment by jean-philippe-martin** [comment]
... so, is it safe to assume that you agree that #867 is good to merge?

**Comment by lbergelson** [comment]
I don't actually understand the all* syntax... If it works and tests pass I'm willing to merge though.  I'm confused why forcing it to 18 isn't enough.  I'm :+1: on this thought if it works empirically and we can understand what's going on later if we have to.

**Comment by jean-philippe-martin** [comment]
Thanks! Forcing guava to 18 isn't enough because guava-jdk5 isn't the same module, so the forcing rule doesn't apply to it.

**Comment by jean-philippe-martin** [comment]
Fixed by #867.
